src/api/app/controllers/webui/user_controller.rb

```diff
@@ -184,6 +189,12 @@ def password_dialog
   end
 
   def change_password
+    if CONFIG[:ldap_mode] == :on
```

Comment:
The save method doesn't change the password. Regarding the API: Good point - will check it!